GVRf/Framework/jni/engine/importer/importer.cpp

```diff
@@ -34,8 +34,9 @@ AssimpImporter* Importer::readFileFromSDCard(std::string str) {
     const char* c_str = str.c_str();
     importer->ReadFile(str,
             aiProcess_JoinIdenticalVertices | aiProcess_FlipUVs);
-    importer->ReadFileFromMemory(c_str, str.size(),
-            aiProcess_JoinIdenticalVertices | aiProcess_FlipUVs, 0);
+    /* No need to load call ReadFileFromMemory as ReadFile is already called */
+    /*importer->ReadFileFromMemory(c_str, str.size(),
+            aiProcess_JoinIdenticalVertices | aiProcess_FlipUVs, 0);*/
     return new AssimpImporter(importer);
```

Comment:
Can you remove the commented codes since they are not needed in future?

And one more concern here, did you try with readFileFromSDCard? I haven't checked it before but currently I am doubting if you should leave ReadFileFromMemory here and comment out the ReadFile( because it is wrong). When I checked "Java_org_gearvrf_NativeImporter_readFileFromSDCard" in importer_jni.cpp, I was seeing that the std::string str for readFileFromSDCard is actually the content of the file. But as you may know, the ReadFile is accepting the file name/path but not the content of the file. 

For this reason, I think in this pull request you may want to either keep ReadFileFromMemory here or change the implementation of readFileFromSDCard in importer_jni.cpp even though I know it is not your fault.   :-)